formwidgets/MLNestedForm.php

```diff
@@ -0,0 +1,195 @@
+<?php namespace Winter\Translate\FormWidgets;
+
+use Backend\FormWidgets\NestedForm;
+use Winter\Translate\Models\Locale;
+use Winter\Storm\Html\Helper as HtmlHelper;
+use ApplicationException;
+use Request;
+
+/**
+ * ML NestedForm
+ * Renders a multi-lingual nestedform field.
+ *
+ * @package rainlab\translate
+ * @author Luke Towers
+ */
+class MLNestedForm extends NestedForm
+{
+    use \Winter\Translate\Traits\MLControl;
+
+    /**
+     * {@inheritDoc}
+     */
+    protected $defaultAlias = 'mlnestedform';
+
+    /**
+     * {@inheritDoc}
+     */
+    public function init()
+    {
+        parent::init();
+        $this->initLocale();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function render()
+    {
+        $this->actAsParent();
+        $parentContent = parent::render();
+        $this->actAsParent(false);
+
+        if (!$this->isAvailable) {
+            return $parentContent;
+        }
+
+        $this->vars['nestedform'] = $parentContent;
+        return $this->makePartial('mlnestedform');
+    }
+
+    public function prepareVars()
+    {
+        parent::prepareVars();
+        $this->prepareLocaleVars();
+    }
+
+    /**
+     * Returns an array of translated values for this field
+     * @return array
+     */
+    public function getSaveValue($value)
+    {
+        $this->rewritePostValues();
+        return $this->getLocaleSaveValue($value);
```

Comment:
I'm curious why you diverged from the previous implementation here. It used this code:
```
return $this->getLocaleSaveValue(is_array($value) ? array_values($value) : $value);
```

There's a reason for this.

Reply:
That code was also taken from the MLRepeater and works well for a list of values (repeater items) but with a nested forms the keys are required. The code above strips away the keys with `array_values($value)` or am I missing something here?